src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -18,6 +18,7 @@ public enum IamAction {
   UPDATE_AUTH_DOMAIN,
   ADMIN_READ_SUMMARY_INFORMATION,
   // datarepo (admin-only actions)
+  ADMIN_TOGGLE_INHERIT_STEWARD,
```

Comment:
I think `SET` is better here than `TOGGLE` since the operation is setting it to a value, not a toggle operation.
```suggestion
  ADMIN_SET_INHERIT_STEWARD,
```

Reply:
I agree set is better, but since we made the sam action toggle, the name has to be toggle here as well so they match

Reply:
Could we change the Sam action name? It isn't being used yet so that seems like it would be safe to do.

Reply:
yeah I like that idea: https://github.com/broadinstitute/sam/pull/1648